apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/GetSchemaTableTests.cs

```diff
@@ -105,12 +116,14 @@ public async Task GetSchemaTable_WithFlagEnabled_ReturnsColumnOrdinals()
                 {
                     var table = reader.GetSchemaTable();
                     // The table also contains the `K` column that is the primary key.
-                    var expectedRowCount = _fixture.RunningOnEmulator ? SchemaTestData.Count() - 1 : SchemaTestData.Count() + 1;
+                    var expectedRowCount = _fixture.RunningOnEmulator
```

Comment:
Presumably the  -1 / +1 is due to two tests not running on the emulator? I think it would be clearer to explain that via another property or something similar. (I haven't mentally mapped out exactly what I'd do, but I'm happy to do it in another PR; no need to change this one.)